### PR TITLE
Author fields & templates; Blog App updates

### PR DESCRIPTION
**Pull request by burieberry**
Changes:
- Add custom isolated, embedded, fitted templates for Author card
- Added experimental field: `FeaturedImage` and use in author and blogpost isolated templates (uses image url, has caption/credit fields)
- Updates from last sprint planning demo: 
	- Use custom fonts only in isolated view;
	- Remove line-clamp from embedded view title and description (this means now cards may be cropped on cards-grid -- will handle cards-grid update separately)
	- Clicking "Create New" from BlogApp links the created card to the blog app by default
- Fix display of authors in Blog App (they did not show up in current state, `isUsed` field property was needed)

Isolated Author:
isolated-author
isolated-small

Embedded Author:
embedded-author

One of the fitted views:
fitted-author

### CODE REVIEW

File: packages/experiments-realm/blog-app.gts [hunk 1]
```diff
@@ -337,10 +339,26 @@ class BlogAppTemplate extends Component<typeof BlogApp> {
...
+      }
       filter.query = {
         filter: {
           on: cardRef,
-          eq: { 'blog.id': this.args.model.id! },
+          any: [
+            { eq: { 'blog.id': this.args.model.id } },
+            { eq: { 'blog.id': relativeTo } },
+          ],
         },
```
Comment (burieberry) [inline packages/experiments-realm/blog-app.gts:362]: This is to cover both relative and absolute urls in the linksTo search.

Comment (lukemelia) [reply packages/experiments-realm/blog-app.gts:362]: @habdelra @ef4 this seems like something we should normalize in our query engine

**Review by lukemelia — APPROVED** [review]
Looks terrific!

**Review by richardhjtan — APPROVED** [review]
Looks really good! 👍

**Review by jurgenwerk — APPROVED** [review]
I tested this locally and it looks good

**Pull request merged**